.github/workflows/release.yml

```diff
@@ -36,11 +36,17 @@ jobs:
 
       - name: Publish to VSCode Marketplace
         if: steps.changesets.outputs.published == 'true'
+        working-directory: ./packages/vscode
         run: |
-          npm i -g vsce
-          cd packages/vscode
-          npm install
-          vsce publish -p ${{ secrets.VSCE_TOKEN }}
+          npm i
```

Comment:
```suggestion
```

Reply:
the same like above, it seems to be necessary